### PR TITLE
PP-5455: GovUkPayEvent changes

### PR DESCRIPTION
**Pull request by oswaldquek**
…R_SETUP_CANCELLED state

## WHAT YOU DID
_A brief description of the pull request:_

## How to test

- How should it be reviewed? 
- What feedback do you need? 
- If manual testing is needed, give suggested testing steps.

### Logging

- [ ] only emit log lines at ERROR level which require immediate attention from a support engineer. These will trigger a zendesk alert.

### Documentation

- [ ] Updated README.md for any of the following ?

* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition

### CODE REVIEW

**Comment by oswaldquek** [comment]
THIS IS A DRAFT PR

File: src/main/java/uk/gov/pay/directdebit/payments/services/GovUkPayEventToPaymentStateMapper.java [hunk 1]
```diff
@@ -9,10 +9,12 @@
 import java.util.Optional;
 import java.util.Set;
 
+import static uk.gov.pay.directdebit.events.model.GovUkPayEventType.PAYMENT_CREATED;
 import static uk.gov.pay.directdebit.events.model.GovUkPayEventType.PAYMENT_SUBMITTED;
 
 public class GovUkPayEventToPaymentStateMapper {
     private static final Map<GovUkPayEventType, PaymentState> GOV_UK_PAY_EVENT_TYPE_TO_PAYMENT_STATE = Map.of(
+            PAYMENT_CREATED, PaymentState.CREATED,
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/directdebit/payments/services/GovUkPayEventToPaymentStateMapper.java]: Doesn’t this also require updating `GovUkPayEventStateGraph` to say that `PAYMENT_CREATED` followed by `PAYMENT_SUBMITTED` is a valid sequence of events?

Comment (oswaldquek) [reply src/main/java/uk/gov/pay/directdebit/payments/services/GovUkPayEventToPaymentStateMapper.java]: I think it makes more sense to do that when I'm implementing the "Make a new payment being inserted into the database save a PAYMENT_CREATED GOV.UK Pay event" acceptance criteria (which will be the next PR)

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/directdebit/payments/services/GovUkPayEventToPaymentStateMapper.java]: OK!

**Pull request merged**